tests/model_forms/tests.py

```diff
@@ -1701,9 +1701,21 @@ class Meta:
             form.errors['author'],
             ['Select a valid choice. That choice is not one of the available choices.']
         )
+        self.assertFalse(form.has_changed())
```

Comment:
As far as I see, it's possible to test the model fields the same way as the others:
```
field = forms.ModelChoiceField(Author.objects.all(), disabled=True)
self.assertIs(field.has_changed('x', 'y'), False)
```